games-strategy/wesnoth/wesnoth-1.18.2.recipe

```diff
@@ -0,0 +1,175 @@
+SUMMARY="Turn-based tactical strategy game with a high fantasy theme"
+DESCRIPTION="Build up a great army, gradually turning raw recruits into \
+hardened veterans. In later games, recall your toughest warriors and form a \
+deadly host whom none can stand against! Choose units from a large pool of \
+specialists, and hand-pick a force with the right strengths to fight well on \
+different terrains against all manner of opposition.
+
+Wesnoth has many different sagas waiting to be played. Fight to regain the \
+throne of Wesnoth, of which you are the legitimate heir... step into the \
+boots of a young officer sent to guard a not-so-sleepy frontier outpost... \
+lead a brutal quest to unite the orcish tribes... vanquish a horde of undead \
+warriors unleashed by a foul necromancer, who also happens to have taken your \
+brother hostage... guide a band of elvish survivors in an epic quest to find \
+a new home... get revenge on the orcs by using the foul art of necromancy...
+
+There are at least two hundred unit types, sixteen races, six major factions, \
+and hundreds of years of history. The world of Wesnoth is absolutely huge and \
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2024 The Battle for Wesnoth Project"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.18/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="5f9f6d6105d48cf614eb06cf72273d62931dcb5d4982d75e851c3fa2fa70553a"
+PATCHES="wesnoth-$portVersion.patchset"
+ADDITIONAL_FILES="
+	wesnoth.rdef.in
+	wesnoth-postinstall.sh
+	"
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/wesnoth-postinstall.sh"
+
+ARCHITECTURES="?all !x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix = $portVersion
+	cmd:wesnoth
+	cmd:wesnothd
+	cmd:wesnoth_campaignd
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_coroutine$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_graph$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_locale$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+#	lib:libdbus_1$secondaryArchSuffix # Optional, for notification module (isn't built either way atm)
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libhistory$secondaryArchSuffix # Optional, for Lua console support
+	lib:libicudata$secondaryArchSuffix
```

Comment:
If ICU is required add a version string to BUILD_REQUIRES like:
`devel:libicuuc$secondaryArchSuffix >= 74`